### ISSUE TITLE
pixelated-maintenance: Present user with better error message if SSL/TLS cert fingerprints do not match

### ISSUE THREAD

**Issue by fbernitt** [pixelated/pixelated-user-agent#390]
When the pixelated-maintenance script is called with a fingerprint as command line argument, e.g.

```
pixelated-user-agent... -lf 1b4f20122cca1733ddfb2e012e27c0cb35428d90 ...
```

The user currently sees a stack trace. We should handle that in the same way that the pixelated-user-agent does it and present a link to our wiki as well as some information about the actual exception.

See service/pixelated/support/error_handler.py:

```
from requests.exceptions import SSLError


def error_handler(excp):
    if excp.type is SSLError:
        print """
            SSL Error: Please check your certificates or read our wiki for further info:
            https://github.com/pixelated-project/pixelated-user-agent/wiki/Configuring-and-using-SSL-Certificates-for-LEAP-provider
            Error reference: %s
            """ % excp.getErrorMessage()
    else:
        raise excp
```

**Comment by re-nobre** [comment]
This is a good idea but it isn't our focus now. 
Our focus now is the migration of Bitmask libraries latest version to Pixelated code.

I'll close this issue for now, it might be reopened in the future if it makes sense.

**Comment by varac** [comment]
i don't agree - we have this problem again (see https://github.com/pixelated/project-issues/issues/144) and it would be super helpful to have more debug information.

**Comment by re-nobre** [comment]
Hey, @varac !
As we closed the issue  pixelated/project-issues#144 and no one reported this problem again, do you think we can close this issue?

**Comment by anikarni** [comment]
Closed due to inactivity
